docs/conf.py

```diff
@@ -520,3 +521,8 @@ def linkcode_resolve(domain, info):
 
 # If false, no index is generated.
 # epub_use_index = True
+
+# -- Options for MyST parser ----------------------------------------------
+myst_heading_anchors = 3
+
+myst_enable_extensions = ["substitution"]
```

Comment:
Not sure why this was added? AFAICT this would be used together with `myst_substitutions` or `substitution` param? https://myst-parser.readthedocs.io/en/latest/configuration.html

Reply:
This was to try out including the Version number in the main text on the docs home page: https://github.com/Parcels-code/Parcels/blob/ac81305934128738c0b1aaea6c65cac5214cb1b5/docs/index.md?plain=1#L11

But I think it might be unnecessary given that we already have the version switcher?

Reply:
Slightly unnecessary - but I think it's nice to double up so that people know for sure which version they're browsing.

Happy to keep